tests/unit/constants.py

```diff
@@ -0,0 +1,5 @@
+FINDING_OKTA_DOMAIN = "https://bit.ly/finding-okta-domain"
+GET_OKTA_API_TOKEN = "https://bit.ly/get-okta-api-token"
+FINDING_OKTA_APP_CRED = "https://bit.ly/finding-okta-app-credentials"
+REPO_URL = "https://github.com/okta/okta-sdk-python"
+GET_OKTA_API_TOKEN = "https://bit.ly/get-okta-api-token"
```

Comment:
Again, curious on where the bit.ly links are coming from.  I would rather see direct links, or non 3rd party provider links